src/azul/service/storage_service.py

```diff
@@ -21,6 +22,7 @@
 AWS_S3_DEFAULT_MINIMUM_PART_SIZE = 5242880  # 5 MB; see https://docs.aws.amazon.com/AmazonS3/latest/dev/qfacts.html
 
 MULTIPART_UPLOAD_MAX_WORKERS = 4
+MULTIPART_UPLOAD_MAX_CONCURRENT_TASKS = 4
```

Comment:
```suggestion
MULTIPART_UPLOAD_MAX_PENDING_PARTS = 4
```

and add a comment explaining what this does. For example, explain what setting this to zero would mean.